### PR TITLE
Final Changes for 2.0

### PR DESCRIPTION
**Pull request by smfsh**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -78,36 +71,37 @@ docker-stop: # Stop containers for this project
...
 	docker-compose -f ${DOCKER_COMPOSE_FILE} down -v
 
 composer-purge:
-	docker exec -i ${PROJECT}-php rm -rf /var/www/webroot/core/*
-	docker exec -i ${PROJECT}-php rm -rf /var/www/webroot/libraries/*
-	docker exec -i ${PROJECT}-php rm -rf /var/www/webroot/modules/contrib/*
-	docker exec -i ${PROJECT}-php rm -rf /var/www/webroot/profiles/contrib/*
-	docker exec -i ${PROJECT}-php rm -rf /var/www/webroot/themes/contrib/*
-	docker exec -i ${PROJECT}-php rm -rf /var/www/vendor/*
+	$(CURDIR)/bin/tool rm -rf /var/www/webroot/core/*
```
Comment (smfsh) [inline Makefile]: These commands are close, but we might need to rethink them. They require the containers to be running, which might not always be the case in the event of a `make destroy`. A user might want to purge all their stuff out in the event, for example, that they have an issue with the php container and the container won't stay running. If this is the case, nothing gets deleted and `make destroy` is not completed.

**Comment by smfsh** [comment]
`COPY webroot /var/www/webroot`

This needs to be re-evaluated. When dockerfiles are poked to determine if a rebuild is necessary, as is the case with `make init` and `make update`, this line causes Docker to rebuild the images any time the environment is new. While this might seem like the preferred behavior, it adds a lot of time to the build process because docker doesn't use its cached images.

Additionally, once these files are written, this image becomes stale until another rebuild. If a user builds their environment on Monday, works on it through Friday without a `make init` or `make update`, when they do a `make export-prod` they will get Monday's code. Beyond this, the code we're copying in here from the `webroot` isn't enough to satisfy the installation as we are not copying in the `config` directory which would be necessary for this site to function on its own.

File: Makefile (hunk 1 above)
Comment (jackinloadup) [reply Makefile]: Agreed, `bin/tool` doesn't solve that problem yet so we still need something else. Likely something that runs alpine:latest and doesn't rely on any images being built.

File: Makefile [hunk 2]
```diff
@@ -78,36 +71,37 @@ docker-stop: # Stop containers for this project
...
-composer-install: # Installs composer packages from composer.lock file
+composer-install: # Installs Composer packages from composer.lock file
 	$(CURDIR)/bin/composer install \
 		  --ignore-platform-reqs \
 		  --no-interaction \
 		  --no-progress
 
+composer-update: # Update all composer managed libraries
+	$(CURDIR)/bin/composer update \
+		--ignore-platform-reqs -vvv
```
Comment (smfsh) [inline Makefile]: Remove super verbose.

**Pull request closed without merging**